parcels/fieldset.py

```diff
@@ -409,21 +403,18 @@ def from_netcdf(
                 for dim, p in paths.items():
                     paths[dim] = cls._parse_wildcards(p, filenames, var)
 
-            # Use dimensions[var] and indices[var] if either of them is a dict of dicts
+            # Use dimensions[var] if its a dict of dicts
```

Comment:
```suggestion
            # Use dimensions[var] if it's a dict of dicts
```

Reply:
done in 9badc73e2c5392e9fd440eb0ef121d1aca26bbbb